### ISSUE TITLE
virtualbox(-with-extension-pack)-np: failed to install because of missing executable

### ISSUE THREAD

**Issue by kkimurak** [ScoopInstaller/Nonportable#71]
## Bug Report

**Package Name:** [`virtualbox-np`](https://github.com/ScoopInstaller/Nonportable/blob/master/bucket/virtualbox-np.json) and [`virtualbox-with-extension-pack-np`](https://github.com/ScoopInstaller/Nonportable/blob/master/bucket/virtualbox-with-extension-pack-np.json)

### Current Behaviour

Because corresponding build flag have been turned to "disable", virtualbox installation does not contain VBoxSDL.exe (an SDL based GUI) for 7.x release.
With this change, the installation stops with an error that cannot create a shim for `VBoxSDL`.
See the upstream changeset : https://www.virtualbox.org/changeset/95176/vbox

### Expected Behaviour

Install succeed with no error reported

### Additional context/output

Here is a terminal output of installation command on current `master` branch ([d5eb9ae](https://github.com/ScoopInstaller/Nonportable/commit/d5eb9aef7927ba00f5eded4f8d3e7b3594affda4))

```shell
> sudo scoop install virtualbox-np
Installing 'virtualbox-np' (7.0.0) [64bit] from nonportable bucket
VirtualBox-7.0.0-153978-Win.exe (104.8 MB)
Linking C:\tools\scoop\apps\virtualbox-np\current => C:\tools\scoop\apps\virtualbox-np\7.0.0
Creating shim for 'VBoxBalloonCtrl'.
Creating shim for 'VBoxBugReport'.
Creating shim for 'VBoxDTrace'.
Creating shim for 'VBoxExtPackHelperApp'.
Creating shim for 'VBoxHeadless'.
Creating shim for 'vbox-img'.
Creating shim for 'VBoxManage'.
Creating shim for 'VBoxNetDHCP'.
Creating shim for 'VBoxNetNAT'.
Creating shim for 'VBoxSDL'.
Can't shim 'VBoxSDL.exe': File doesn't exist.  
```

### Possible Solution

Just remove `VBoxSDL` from shim target

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** 7.2.6

**Additional software:** N/A

#### Scoop Configuration

```json
{
  "lastupdate": "2022-10-12T12:44:39.0379003+09:00",
  "SCOOP_REPO": "https://github.com/lukesampson/scoop",
  "SCOOP_BRANCH": "master",
  "debug": true
}
```

**Comment by kkimurak** [comment]
I have already confirmed the solution works and ready to submit PR.

**Comment by kkimurak** [comment]
Will be fixed by #72 and #73 . Closing
